doc/source/user_guide/window.rst

```diff
@@ -158,6 +158,7 @@ By default the labels are set to the right edge of the window, but a
 
 
 This can also be applied to datetime-like indices.
+
 .. versionadded:: 1.3
 .. ipython:: python
```

Comment:
Not sure if it will be needed for sphinx, but might add an empty line between the versionadded and the ipython block as well

Reply:
Looked ok when I built locally, but agreed might as well keep empty line between directives

Reply:
Yeah, it's not always clear when sphinx expects one and when not .. ;) 
In any case around paragraphs (as you did in the first commit) it's needed.